softwerkskammer/lib/groups/views/get.pug

```diff
@@ -30,7 +30,12 @@ block content
         h1 #{group.longName} #{' '}
           small #{group.type}
       p
-        strong #{t('general.address')}: #{' '}
+        if (group.contactTheOrganizers && accessrights.canParticipateInGroup(group))
```

Comment:
Alle derartigen Berechtigungs-Entscheidungen sollten in den Accessrights gesammelt werden, damit sie leicht zentral änderbar sind.

Reply:
Das eine ist eine Gruppeneigenschaft, das andere Berechtigung

Reply:
Was meinst Du damit? Die Logik, dass diese Gruppeneigenschaft so vorliegen muss, gehört doch zur Berechtigung dazu. Dieses Wissen hätte ich gern an einer Stelle in den accessrights.

Reply:
Ach so. Für mich ist das eine ein Anzeigeproperty, wohingegen Sicherheitssachen insbesondere Eigenschaften des Users prüfen

Reply:
Hm, verstehe, ok. Dann würde ich dafür plädieren, dass es zumindest eine eigene accessrights-Methode für den Berechtigungsteil des Ausdrucks gäbe. Denn die Logik soll sich ja dann mal ändern. Das fände ich schon nicht schlecht, wenn die dann gleich zentral wäre.

Reply:
Mit "Berechtigungsteil des Ausdrucks" ist dann `goup.contactTheOrganizers` (bzw. ein besserer Name, wenn wir einen gefunden haben ;-) ) gemeint?

Reply:
Ich habe jetzt mal den gesamten Ausdruck in die neue Methode `canContactTheOrganizers` in `accessrights.js` gepackt. Erschien mir recht stimmig, dass zusammenzufassen. Ich kann das aber auch wieder auseinander ziehen.